defaults/main.yml

```diff
@@ -82,7 +83,8 @@ wimpy_alarm_metrics:
 # - new_asg: old instances stay alive, new stack is created
 wimpy_deploy_strategy: "rolling_asg"
 
-wimpy_dns_ttl: 300 # DNS record Time To Live
+wimpy_dns_ttl: 60 # DNS record Time To Live
+wimpy_new_dns_weight: 10 # DNS record weight
```

Comment:
Any number is valid, but 10 is small enough to be easy to work with it, and also it allows you to do granular weights like deploying a new dns with weight 1 to send the 10% of your traffic.